logging/src/logging.cc

```diff
@@ -85,7 +86,9 @@ namespace leatherman { namespace logging {
         // Note that this creates a locale that's not usable for testing, as it
         // only includes paths for install locations. This is intentional, to avoid leaving
         // searching paths that have unknown permissions.
-        dst.imbue(lth_locale::get_locale(locale, domain, {}));
+        if (use_locale) {
+            dst.imbue(lth_locale::get_locale(locale, domain, {}));
```

Comment:
I think a good "leatherman breaking change" would be disentangling locale and logging setup, since (in theory) an app might want to set up locales for translation without doing any logging at all